templates/home.html

```diff
@@ -12,7 +12,7 @@
                 <div class="home--content-inner">
                   <h1>{% trans "Promo &amp; Sale" %}</h1>
                   <h2>{% trans "form the Northpole" %}</h2>
-                    <a class="btn btn-secondary home__button" href="/products/category/default-1/">{% trans "Shop now" %}</a>
+                    <a class="btn btn-secondary home__button" href="{% if featured_categories %}{{ featured_categories.0.get_absolute_url }}{% endif %}">{% trans "Shop now" %}</a>
```

Comment:
Those homepage blocks aren't the same, if I used `for-loop` here I'd need to do custom checks whether to display full-width block or half-width.